frontend/app/services/GuardianContentService.scala

```diff
@@ -101,8 +114,23 @@ trait GuardianContent {
     }
   }
 
+  def membersOnlyContentQuery(page: Int): Future[ItemResponse] = {
+    val itemQuery = ItemQuery("extra")
+      .showFields("all")
+      .showElements("all")
+      .showTags("all")
+      .pageSize(100)
+      .page(page)
+      .tag("type/quiz | tone/extraoffers")
+    //todo: filter response for member access flag
```

Comment:
This filter will have to be done on the results I think rather than in the query.